DiceKeys/Views/Screens/EditableDiceKeyView/KeyboardCommands.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  KeyboardCommands.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 22/01/21.
+//
+
+import SwiftUI
+
+struct KeyboardCommandsModel: Identifiable {
+    var id: String
+    var key: KeyEquivalent
+}
+
+struct KeyboardCommands: Commands {
```

Comment:
A comment would be useful here to explain how this piece fits to the others.

Is this a phantom menu?  Can users see it or is it hidden?
How is it hooked up?

Reply:
Yes, there is menu with blank string at the status bar. Was investigating how to hide command menu for same.